tests/test_admin.py

```diff
@@ -171,12 +172,66 @@ def test_get_queryset_filters_by_content_id(self):
         )
 
 
+class MenuItemAdminVersionLocked(CMSTestCase, UsefulAssertsMixin):
+
+    def setUp(self):
+        # who is author..
```

Comment:
What is the following comment for? # who is author

Reply:
I think it was for the version of the factory. But taking a look at how another version factory looked like it looks the same. So I deleted this.